ts/IterableInboxMessageDisplay.tsx

```diff
@@ -70,17 +71,27 @@ const IterableInboxMessageDisplay = ({
    })
 
    const handleHTMLMessage = (event: any) => {
-      if(event.nativeEvent.data === 'iterable://delete') {
+      let url = event.nativeEvent.data
+      if(url === 'iterable://delete') {
```

Comment:
Not needed for this PR but we should try to be more consistent with syntax, like spacing between an `if` and the condition

Reply:
Tip: If you are using VS Code, select the code section you edited and use `Opt + Shift + F`. This will indent the code

Reply:
I went ahead and formatted all the inbox files